src/vivarium/component.py

```diff
@@ -244,15 +262,15 @@ def initialization_requirements(
         return []
 
     @property
-    def population_view_query(self) -> str | None:
+    def population_view_query(self) -> str:
         """Provides a query to use when filtering the component's `PopulationView`.
 
         Returns
         -------
             A pandas query string for filtering the component's `PopulationView`.
-            Returns `None` if no filtering is required.
+            Returns an empty string if no filtering is required.
         """
-        return None
+        return ""
```

Comment:
This seems like a big change based on validation for empty strings vs None. What is the reason for this change or was the type hinting just wrong?

Reply:
Still working on this (the PR is in draft)

Reply:
I think it's fine, though. It did indeed get flagged by mypy but all of the usage of the query are handling strings

Reply:
I guess there's a chance that some downstream code is assuming this could be None. I can handle it if we think that's actually the case. but vph isn't using this property at all and everything in vivarium handles it as a str. (all of the defaults, e.g. are `""`, not `None`)